### ISSUE TITLE
Server Cleaner not working

### ISSUE THREAD

**Issue by Lemunu** [DeadBread76/Raid-Toolbox#64]
it says attack running but nothing is happening

**Comment by sewek333** [comment]
Yes I have the same problem 'user' do not fall on my server come have the correct invitation

**Comment by hidden-r4d** [comment]
From what I've seen (using the latest version), the server cleaner *does* work, but takes a moment to get started.
